tests/landIce/FO_GIS/input_fo_gis_analysis_stiffening_memT.yaml

```diff
@@ -73,9 +73,8 @@ ANONYMOUS:
     Workset Size: -1
     Method: Extruded
     Number Of Time Derivatives: 0
-    Cubature Degree: 1
+    Cubature Degree: 3
```

Comment:
We typically use Cubature Degree: 4 for Wedges. I found that sometime with degree 3 it takes more iterations to converge.

Reply:
Ok, I can sed/replace 4 everywhere. I found 3 somewhere, so I used that, but I can just use 4 everywhere. What about the lateralside quadrature order? With tets, we used 3. Is 4 enought?

Reply:
I think 4 is enough. The reason why we need much higher quadrature for prism than tets for the volume integration is that the viscosity depends  (nonlinearly) only on the grad of the velocity, which is piece-wise constant for Tets so 1 quad point is enough.

Reply:
Well, the lateralside BC also has the immersedRatio quantity, which is something like `z_s/H`, which is also nonlinear.

Reply:
agreed, but is more or less equally bad both for triangles and quads

Reply:
Fixed!